smart-contracts/erc20/assembly/exportable/erc20Core.ts

```diff
@@ -0,0 +1,40 @@
+import { Args } from "@massalabs/as-types";
+import { Address } from "@massalabs/massa-as-sdk";
+
+export function allowance(_args: StaticArray<u8>): StaticArray<u8> {
+  const args = new Args(_args);
+  const owner = args.mustNext<Address>("owner");
+  const spender = args.mustNext<Address>("spender");
+  FungibleToken.allowance(owner, spender).serialize();
```

Comment:
where is FungibleToken defined (I see it in the tests but not in the code here)

Reply:
It's a global variable. This trick is used to export the functions.

Reply:
but this global variable is exported in the test file?

Reply:
Yes, it's to demonstrate what the end user will need to do to use our standard, directly within the unit tests. The export is supposed to be handled at the usage level, not on the standard side.